src/control/PlanarArmController.h

```diff
@@ -104,12 +110,18 @@ class PlanarArmController {
 	 * @param targetVel The target y velocity.
 	 * @return The new command, which is the new joint positions.
 	 */
-	void set_y_vel(robot::types::datatime_t currTime, double targetVel) {
+	void set_y_vel(robot::types::datatime_t currTime, double targetVel,
+				   robot::types::DataPoint<navtypes::Vectord<N>> jointPos) {
```

Comment:
Same as above, a) we should update the doxy docs and b) don't think there's a point in making this a `DataPoint`.